cli/config.go

```diff
@@ -141,7 +209,27 @@ func (c Config) ExecPath() string {
 }
 
 func (c Config) buildPath(base string) string {
-	p := filepath.FromSlash(c[EXECLINE].(string))
+	execpath := c[APPPATH].(string)
+	//empty app path defaults to looking for DAISY Pipeline app in the PATH
```

Comment:
Notes to self (possibly add these as comments):
- We choose not to use "DAISY Pipeline" as the default config value, because this would result in a "provided app path was not found" warning if no app is found.
- Having no default value at all would also have been a possibility, because the field will be set in the config.yml that we ship with the app anyway, and in the general case we can assume there is no app. However it doesn't hurt to add it either, especially because the app will not be searched for anyway when `host`, `port` and `ws_path` are set.